Libraries/xcworkspace/Tools/dump_xcworkspace.cpp

```diff
@@ -235,7 +235,7 @@ main(int argc, char **argv)
         std::sort(schemes.begin(), schemes.end(),
                 [](xcscheme::XC::Scheme::shared_ptr const &a, xcscheme::XC::Scheme::shared_ptr const &b) -> bool
                 {
-                    return ::strcasecmp(a->name().c_str(), b->name().c_str()) < 0;
+                    return ::strcmp(a->name().c_str(), b->name().c_str()) < 0;
```

Comment:
Yeah, this is just a debugging tool so doesn't really matter, and avoids the extra lines to be insensitive.